spec/components/card_component_spec.rb

```diff
@@ -8,6 +8,6 @@
 
     render_inline(component)
 
-    expect(rendered_component).to have_css("div[data-controller='navbar'][data-navbar-target='menu-state']")
+    expect(page).to have_css("div[data-controller='navbar'][data-navbar-target='menu-state']")
```

Comment:
I got a deprecation warning while running specs, so did a bit of tidying while I was here.

```
DEPRECATION WARNING: `rendered_component` is deprecated and will be removed in v3.0.0. Use `page` instead. 
```